src/main/scala/cromwell/Main.scala

```diff
@@ -293,21 +289,30 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
         |  via this sub-command and the 'validate' subcommand should
         |  be used for full validation
         |
-        |highlight <WDL file> <html|console>
+        |validate <WDL file> <backend type>
+        |
+        |  Performs full validation of the WDL file including syntax
+        |  and semantic checking
+        |
+        |inputs <WDL file> <backend type>
+        |
+        |  Print a JSON skeleton file of the inputs needed for this
+        |  workflow.  Fill in the values in this JSON document and
+        |  pass it in to the 'run' subcommand.
+        |
+        |highlight <WDL file> <backend type> <html|console>
         |
         |  Reformats and colorizes/tags a WDL file. The second
         |  parameter is the output type.  "html" will output the WDL
         |  file with <span> tags around elements.  "console" mode
         |  will output colorized text to the terminal
         |
-        |server
-        |
-        |  Starts a web server on port 8000.  See the web server
-        |  documentation for more details about the API endpoints.
+        |Supported backend types: local, sge, jes
```

Comment:
it would be preferable to generate this from `BackendType.values()` just in case we receive a large code contribution in the near future that introduces more.  :smile: